controller/inviteController.js

```diff
@@ -33,48 +22,39 @@ const create = (req, res) => {
 const getById = (req, res) => {
 
     const inviteId = req.params.id;
```

Comment:
You have to check if `inviteId` is a valid `ObjectId`. Check `ValidationUtil` for that. Also why are you not checking for `user`? Makes no sense for anyone to be able to retrieve anyone's invite.